src/os/bluestore/BitAllocator.h

```diff
@@ -322,6 +322,8 @@ class BitMapZone: public BitMapArea{
   static int64_t total_blocks;
   static void incr_count() { count++;}
   static int64_t get_total_blocks() {return total_blocks;}
+
+  using BitMapArea::is_allocated;
```

Comment:
IMHO one can simply remove 
virtual bool is_allocated(ExtentList *blocks, int64_t num_blocks, int blk_off)
to fix the warning. Looks like it's not used..